src/constants/typography.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components"
+import "./typefaces.css"
+
+const fonts = {
+  workSans: `'Work Sans', sans-serif`,
+  futura: `'Futura', sans-serif`,
+  futuraBold: `'Futura Bold', sans-serif`,
+  futuraHeavy: `'Futura Heavy', sans-serif`,
+}
+
+const fontSizes = {
+  h1: `calc(40px + (90 - 40) * ((100vw - 320px) / (1440 - 320)))`,
+  h2: `calc(30px + (50 - 30) * ((100vw - 320px) / (1440 - 320)))`,
+  h3: `calc(25px + (28 - 22) * ((100vw - 320px) / (1440 - 320)))`,
+  p: `calc(14px + (16 - 14) * ((100vw - 320px) / (1440 - 320)))`,
+}
+
+const textStyles = {
+  uppercase: "uppercase",
+}
+
+const fontWeights = {
+  normal: `400px`,
+  bold: `700px`,
+  heavy: `900px`,
+}
+
+const H1 = styled.h1`
+  font-size: ${fontSizes.h1};
+  font-family: ${fonts.futuraBold};
+  font-weight: ${fontWeights.bold};
+  text-transform: ${textStyles.uppercase};
+  line-height: 60px;
```

Comment:
This might stand to scale like font size, but usually that only applies to body font – I would check with Riha here